optuna_integration/_lightgbm_tuner/optimize.py

```diff
@@ -494,8 +494,7 @@ def sample_train_set(self) -> None:
 
     def tune_feature_fraction(self, n_trials: int = 7) -> None:
         param_name = "feature_fraction"
-        param_values = [0.4 + 0.6 * i / (n_trials - 1) for i in range(n_trials)]
-
+        param_values = cast(list, np.linspace(0.4, 1.0, n_trials).tolist())
```

Comment:
This change was to go back to the original implementation.
https://github.com/optuna/optuna-integration/pull/184/files